go/src/koding/kites/config/konfig.go

```diff
@@ -170,6 +170,7 @@ type Konfig struct {
 	PublicBucketRegion string `json:"publicBucketRegion,omitempty"`
 
 	LockTimeout time.Duration `json:"lockTimeout,omitempty"`
+	NoMetrics   bool          `json:"noMetrics,string,omitempty"`
```

Comment:
Ok, now I remember why I did it that way - the configuration object merging relies on zero values - you can't have a default bool value which is not a zero value. There are two configuration values -  a default one and the one read from bolt. When they are merged into one, the effective field has the field's value of the object, which was non-zero.

That's why when default bool is true, during merging it will always be true. For now it's easier to stick with default values being zero values for bool types, instead of rewriting merging.